draft-ietf-ipsecme-rfc8229bis.xml

```diff
@@ -527,7 +527,7 @@ Length|ESP packet ->
           subsequent messages that cannot be parsed as valid IKE messages or
           ESP messages with known SPIs, or if the authentication check for an
           IKE message or ESP message with a known SPI fails.  Implementations SHOULD NOT
-          tear down a connection if only a single ESP message has an unknown
+          tear down a connection if only a few consecutive ESP packets has an unknown
           SPI, since the SPI databases may be momentarily out of sync.  If
```

Comment:
```suggestion
          tear down a connection if only a few consecutive ESP packets have unknown
          SPIs, since the SPI databases may be momentarily out of sync.  If
```